types/node/net.d.ts

```diff
@@ -18,7 +18,7 @@ declare module 'net' {
     import * as dns from 'node:dns';
     type LookupFunction = (hostname: string, options: dns.LookupOneOptions, callback: (err: NodeJS.ErrnoException | null, address: string, family: number) => void) => void;
     interface AddressInfo {
-        address: string;
+        address: string | number; // number since v18 and string for all previous versions
```

Comment:
This shouldn't be `string|number`.
In `@types/node` 18.x version, this should be just `number`.
In `@types/node` 17.x version, this should be just `string`.

Reply:
@yoursunny Then it would be impossible to support both v16 and v18 in the same code base.
In my case, it's a popular NPM library that should work with all current LTS.

Reply:
This should be on the `family` line, not the `address` line, right?